src/views/List.jsx

```diff
@@ -34,6 +35,9 @@ export function List({ data, listToken, show }) {
 					<form className="listContainer">
 						<div className="search">
 							<i className="fas fa-search search-icon"></i>
+							<label htmlFor="search-input" hidden>
```

Comment:
Hi @naura1835,
I see that you're using the `hidden` attribute for the label. This will cause the label is hidden completely, not only visually, but also from screen readers.

Although it's not a best practice, if you want to hide the label text, you can do so using CSS class to hide it. Alternatively, you can use `aria-label` in the input.
Please refer to this tutorial article from W3C about [hiding label text](https://www.w3.org/WAI/tutorials/forms/labels/#hiding-label-text).

Reply:
i changed it to use the aria-label